dialoguekit/connector/dialogue_connector.py

```diff
@@ -117,14 +117,18 @@ def register_agent_utterance(
             self._user.receive_utterance(annotated_utterance)
 
     def register_user_feedback(
-        self, utterance_feedback: UtteranceFeedback
+        self, feedback: BinaryFeedback, utterance_id: str
```

Comment:
Nit: Mention here that currently only binary feedback is accepted (BinaryFeedback.{NEGATIVE|POSITIVE}).

Reply:
Done